include/seastar/util/backtrace.hh

```diff
@@ -84,18 +84,16 @@ public:
     using vector_type = boost::container::static_vector<frame, 64>;
 private:
     vector_type _frames;
-    size_t _hash;
-    char _delimeter;
+    size_t _hash = 0;
 private:
     size_t calculate_hash() const noexcept;
 public:
-    simple_backtrace(vector_type f, char delimeter = ' ') noexcept : _frames(std::move(f)), _hash(calculate_hash()), _delimeter(delimeter) {}
-    simple_backtrace(char delimeter = ' ') noexcept : simple_backtrace({}, delimeter) {}
+    simple_backtrace(vector_type f) noexcept : _frames(std::move(f)), _hash(calculate_hash()) {}
+    simple_backtrace() noexcept = default;
```

Comment:
they were added as part of the `operator<<` based formatter, so i dropped them. but yeah, since they are part of the public interface, they are preserved. but in the latest revision, i marked them deprecated in favor of the constructors which do not accept the `delimeter` parameter.